### DISCUSSION TITLE
Installare app HA su firestick

### DISCUSSION THREAD

**Discussion by vinny74man** [indomus/forum#1854] [HUB: Home Assistant]
Ciao a tutti come da titolo sapete se è possibile installare l'app di HA su una firestick?
Ho provato a cercare nell'app store della stick ma non riesco a trovarlo.

Grazie

**Comment by snakuzzo** [comment]
Se vuoi provare...abilita la modalità sviluppatore sulla firestick, scarica l'apk dell'app (cerca su google) e fai il sideload con adb

**Reply by vinny74man** [reply]
Ok Grazie, proverò
